src/services/opportunityService.ts

```
@@ -0,0 +1,204 @@
+
+  if (error) {
+    console.error('Error creating opportunity:', error);
+    throw error;
+  }
+
+  return data;
+};
+
+export const updateOpportunity = async (id: string, opportunity: Partial<Omit<Opportunity, 'id' | 'createdAt' | 'updatedAt'>>) => {
+  // Convert from camelCase to snake_case for the database
+  const dbOpportunity: any = {};
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)